.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -7,6 +7,25 @@ labels: kind/bug
 
 <!-- Please use this template while reporting a bug and provide as much info as possible. Thanks!-->
 
+**Environment**:
+- KubeEdge version(e.g. `keadm version -o json`):  
+
+**CloudSide Environment**:
+- cloudcore version (e.g. `cloudcore --version`):
```

Comment:
Remove this line? We need a unify version of kubeedge in cloud and edge side.

Reply:
done